org.keshavnrj.wonderwall.yaml

```diff
@@ -0,0 +1,46 @@
+app-id: org.keshavnrj.wonderwall
+base: io.qt.qtwebkit.BaseApp
+base-version: '5.11'
+runtime: org.kde.Platform
+runtime-version: '5.11'
+sdk: org.kde.Sdk
+command: wonderwall
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=host
+  - --filesystem=xdg-download
+  - --device=dri
+  
+  - --talk-name=org.gtk.vfs
+  - --talk-name=org.gtk.vfs.*
+  - --talk-name=org.freedesktop.FileManager1
+
+  #- --filesystem=/var/lib/lightdm-data:rw
+
+  - --filesystem=xdg-run/dconf
+  - --filesystem=~/.config/dconf:ro
+  - --talk-name=ca.desrt.dconf
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
+  
+  - --talk-name=org.elementary.Contractor
+  - --system-talk-name=org.freedesktop.Accounts
+  
+modules:
+  - name: ww
+    buildsystem: qmake
+    config-opts:
+      - QMAKE_STRIP=
+      - DEFINES+=USE_DBUS
+      - INCLUDEPATH+="${FLATPAK_DEST}/include/QtWebKitWidgets"
+      - INCLUDEPATH+="${FLATPAK_DEST}/include/QtWebKit"
+      - LRELEASE=lrelease
+    build-options:
+      env:
+        QMAKEPATH: "/app/lib"
+    sources:
+      - type: archive
+        url: "http://ktechpit.com/USS/ww_flathub_build.zip"
```

Comment:
and one last thing: an unversioned archive (ie the version isn't the filename) is a source of trouble because if there is a new version, the build will break.

Reply:
Will try to fix this in upcoming PR.

Thanks a lot for all the suggestions. I am actually very new to flatpak.